content/plugins/s3.md

```diff
@@ -27,7 +27,7 @@ Use the S3 plugin to upload files and build artifacts to an S3 bucket. The follo
 * **target** - target location of files in your S3 bucket
 * **recursive** - if true, recursively upload files
 
-The following is a sample Slack configuration in your .drone.yml file:
+The following is a sample S3 configuration in your .drone.yml file:
```

Comment:
https://github.com/KyleOndy/docs/blob/1d098910253f0b2306ecf44aefb92bc2ee530e54/content/plugins/s3.md#this-document-is-automatically-generated-please-do-not-edit

;)